tests/resources/Harbor-Util.robot

```diff
@@ -109,4 +109,83 @@ Toggle Admin Priviledges For User
     \   Run Keyword If  ${status}  Set Test Variable  ROW_NUMBER  ${idx}
     \   Exit For Loop If  ${status}
     
-    Click Element  css=body > div.container-fluid.container-fluid-custom.ng-scope > div > div > div > list-user > div > div > div.pane > div.sub-pane > div.table-body-container > table > tbody > tr:nth-child(${ROW_NUMBER}) > td:nth-child(4) > toggle-admin > button.btn.btn-danger.ng-binding
\ No newline at end of file
+    Click Element  css=body > div.container-fluid.container-fluid-custom.ng-scope > div > div > div > list-user > div > div > div.pane > div.sub-pane > div.table-body-container > table > tbody > tr:nth-child(${ROW_NUMBER}) > td:nth-child(4) > toggle-admin > button.btn.btn-danger.ng-binding
```

Comment:
This is only basic selectors. They can be improved and we need to get Harbor UI team to provide IDs ASAP